samples/user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBListFragment.kt

```diff
@@ -37,6 +38,10 @@ class PBListFragment : Fragment() {
 
         setAnimationText()
 
+        exitTransition = MaterialSharedAxis(MaterialSharedAxis.Z, /* forward= */ true)
+        reenterTransition = MaterialSharedAxis(MaterialSharedAxis.Z, /* forward= */ false)
+
+
```

Comment:
@ashnohe, was wondering if it should be enter and return OR exit and reenter Transitions? the [docs](https://m2.material.io/develop/android/theming/motion#shared-axis) has exit and reenter as the transitions for FragmentA and enter and return transitions for fragmentB?

Reply:
PBListFragment is Fragment A so exit and reenter are correct.